pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -257,14 +258,15 @@ def setUpClass(cls):
         super(SyncValidFeedTestCase, cls).setUpClass()
         body = _gen_rpm_repo_body()
         body['importer_config']['feed'] = _VALID_FEED
-        cls.attrs_iter = (create_repository(cls.cfg, body),)  # see parent cls
+        cls.repo = create_repository(cls.cfg, body)
```

Comment:
Ok, gotcha. I'll change the test back to using that attribute name then.

Michael